rest_framework/fields.py

```diff
@@ -1414,11 +1411,8 @@ def to_internal_value(self, data):
     def to_representation(self, value):
         if value in ('', None):
             return value
-
-        if isinstance(value, (IntegerChoices, TextChoices)) and str(value) != \
-                str(value.value):
+        if isinstance(value, Enum) and str(value) != str(value.value):
```

Comment:
this could be considered as a direct python way alongside the django built options

Reply:
It's important to note that Django's `IntegerChoices` and `TextChoices` are subclasses of `Enum` under the hood. So while it definitely isn't obvious on first glance, the replacement code for the `Enum` class/subclasses covers the previous cases.

Reply:
Yes, that's what I think.Django's `IntegerChoices` and `TextChoices` are subclasses of Enum. 

I think when I replaced `if isinstance(data, (IntegerChoices, TextChoices)) and str(data) !=
str(data.value):` with  ` if isinstance(data, Enum) and str(data) != str(data.value):`, django-rest-framework will still support django compatibility feature `IntegerChoices` and  `TextChoices`.

And at the same time, we also make our framework available in supporting python's native IntEnum in multiple python version.

I believe this is a win-win solution.